src/bootstrap_io.py

```
@@ -0,0 +1,208 @@
+    finally:
+        try:
+            temporary.unlink()
+        except FileNotFoundError:
+            pass
+
+
+def committed_bytes(path: Path, captured: bytes) -> None:
+    """Require captured policy bytes to equal the clean HEAD blob."""
+    if not SYSTEM_GIT.is_file() or SYSTEM_GIT.is_symlink():
+        fail("the system Git verifier is unavailable")
+    repository = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)